src/Sfa.Tl.ResultsAndCertification.Application/Services/AdminPostResultsService.cs

```diff
@@ -33,33 +36,15 @@ public async Task<bool> ProcessAdminOpenPathwayRommAsync(OpenPathwayRommRequest
 
             DateTime utcNow = _systemProvider.UtcNow;
```

Comment:
utcNow is only used inside the CreatePathwayRequest method. Why not just getting it inside the method instead of passing it as a parameter?